lib/Language/Slicer/Resugar.hs

```diff
@@ -265,6 +262,16 @@ instance Resugarable Value where
         = resugarError ("Cannot resugar store labels")
 
 
+instance Resugarable Outcome where
+    resugarM OHole = return RHole
+    resugarM (ORet v) = resugarM v
+    resugarM (OExn v) = do e <- resugarM v
+                           return (RRaise e) 
+    resugarM OStar
+        = resugarError ("Don't know how to desugar stars. " ++
```

Comment:
Argh, I managed to do this in "references".  I think I've finally managed to merge "references" into "jrc-outcomes" so they can be merged if everything is happy.